packages/core/src/components/step-by-step/step-by-step.css

```diff
@@ -0,0 +1,78 @@
+
+.hds-step-by-step__container {
+  position: relative;
+}
+
+.hds-step-by-step__steps-container {
+  counter-reset: item;
+  margin: 0;
+  padding: 0;
+}
+
+.hds-step-by-step__steps-container li {
```

Comment:
Instead of `li` we can use `.hds-step-by-step__step-item`. I'll make the improvement.